scripts/install.sh

```diff
@@ -104,16 +104,20 @@ function setupInstallationPath(){
   else
     einfo "Detected Terraform v$TF_VERSION"
   fi
-  TF_MINOR_VERSION=$(echo $TF_VERSION| cut -f2 -d.)
-  if [ $TF_MINOR_VERSION -lt 12 ]
+
+  TF_PATCH_VERSION=$(echo $TF_VERSION | cut -f3 -d.)
+  TF_MINOR_VERSION=$(echo $TF_VERSION | cut -f2 -d.)
+  TF_MAJOR_VERSION=$(echo $TF_VERSION | cut -f1 -d.)
+
+  if [ $TF_MINOR_VERSION -lt 12 ] && [ $TF_MAJOR_VERSION -eq 0 ]
   then
     eerror "OpenAPI Terraform provider no longer supports versions of Terraform less than v0.12.0"
     exit 1
-  elif [ $TF_MINOR_VERSION -eq 12 ]
+  elif [ $TF_MINOR_VERSION -eq 12 ] && [ $TF_MAJOR_VERSION -eq 0 ]
   then
     einfo "Installing provider based on Terraform v0.12.* plugin installation instructions: https://www.terraform.io/docs/plugins/basics.html#installing-plugins"
     TF_PROVIDER_PLUGIN_NAME="${TF_PROVIDER_BASE_NAME}${PROVIDER_NAME}_v${PLUGIN_VERSION}"
-  elif [ $TF_MINOR_VERSION -ge 13 ]
+  elif (([ $TF_MINOR_VERSION -ge 13 ] && [ $TF_MAJOR_VERSION -eq 0 ]) || [ $TF_MAJOR_VERSION -ge 1 ])
```

Comment:
I believe this statement here can be simplified now and just be the default if the user is using tf versions >= 0.13. Thoughts?

Reply:
Simplicity ain't a bad thing. Do you mean converting the `elif` into an `else` and dropping the conditions altogether? I think that will function the same, and I can update that line if that's what you're suggesting.

Reply:
Yep, updating the elif to else would do it. Thx